oaknational/Oak-Web-Application
# Feat/lesq 1046/threads styling wrap up

## Description

Music year: 2011

- Replace radio tile component with oakRadioTile from components library

## How to test

1. Go to https://deploy-preview-2795--oak-web-application.netlify.thenational.academy/teachers/programmes/english-primary-ks1/units
3. Click on thread filters on the right
4. You should see the list of units updating on selection
5. Navigating with a keyboard should apply filters as they are moved through, without reloading the page
6. Look and behaviour should match production
